src/Header/SetCookie.php

```diff
@@ -572,6 +611,24 @@ public function isSessionCookie()
         return ($this->expires === null);
     }
 
+    /**
+     * @return string|null
+     */
+    public function getSameSite()
+    {
+        return $this->sameSite;
+    }
+
+    /**
+     * @param string $sameSite
+     * @return SetCookie
+     */
+    public function setSameSite($sameSite)
+    {
+        $this->sameSite = $sameSite;
```

Comment:
I believe we should check here if we set the correct value, and throw exception if not?

Reply:
Sure :-)